src/components/keyboard/Write.tsx

```diff
@@ -1,16 +1,16 @@
 import React, {useRef, useState} from 'react';
 import Keyboard from "react-simple-keyboard"
 import "react-simple-keyboard/build/css/index.css";
+import {defaultLayout} from "../../defaults/Layouts";
 
-type Props = {
-}
+type Props = {}
 export default function Write(props: Props) {
-const keyboardRef = useRef(null)
+    const keyboardRef = useRef(null)
     const [input, setInput] = useState("")
     const [layout, setLayout] = useState("default")
 
-    function onChange(input: any) {
-        setInput(input);
+    function onChange(writtenText: any) {
```

Comment:
```suggestion
    function onChange(event: React.ChangeEvent<HtmlInputElement>) {
```
or maybe ChangeEvent without React, since it is used in a html element rather than a react component